### PR TITLE
[CI] Temporarily disable OCL FPGA Emulator

### PR DESCRIPTION
**Pull request by alexbatashev**
FPGA emulator causes crashes on SYCL runtime initialization.

### CODE REVIEW

File: devops/actions/llvm_test_suite/action.yml [hunk 1]
```diff
@@ -83,6 +83,11 @@ runs:
       if [ -e /runtimes/oneapi-tbb/env/vars.sh ]; then
         source /runtimes/oneapi-tbb/env/vars.sh;
       fi
+      # TODO remove workaround of FPGA emu bug
+      mkdir icd
```
Comment (bader) [inline devops/actions/llvm_test_suite/action.yml:87]: Shouldn't we check if `icd` directory exists already?
```suggestion
      mkdir -p icd
```

**Review by bader — APPROVED** [review]
Do I understand it correctly that even if we bypass sycl-ls problem and re-enable FPGA emulator environment, there will be sporadic failures of llvm-test-suite tests?

**Comment by alexbatashev** [comment]
> Do I understand it correctly that even if we bypass sycl-ls problem and re-enable FPGA emulator environment, there will be sporadic failures of llvm-test-suite tests?

That's right, the tests will just fail later.

**Comment by bader** [comment]
Any ideas what might trigger this issue? I see that FPGA emulator hasn't been change for weeks - https://github.com/intel/llvm/blob/sycl/devops/dependencies.json, but the issue appeared quite recently.

**Comment by yanfeng3721** [comment]
Hi @bader , is there any plan to enable oclfpgaemu back for CI testing? 
BTW, will there be an oclfpgaemu CI testing similar as the current oclcpu CI tesing "Linux / OCL x64 LLVM Test Suite" in the future?
```
default_selector()      : gpu, Intel(R) Level-Zero, Intel(R) UHD Graphics [0x9bca] 1.3 [1.3.24278]
  accelerator_selector()  : No device of requested type available. -1 (PI_ERRO...
  cpu_selector()          : cpu, Intel(R) OpenCL, Intel(R) Core(TM) i7-10710U CPU @ 1.10GHz 3.0 [2022.14.8.0.04_1[600](https://github.com/intel/llvm/actions/runs/3245321030/jobs/5323539661#step:8:621)00]
  gpu_selector()          : gpu, Intel(R) Level-Zero, Intel(R) UHD Graphics [0x9bca] 1.3 [1.3.24278]
  custom_selector(gpu)    : gpu, Intel(R) Level-Zero, Intel(R) UHD Graphics [0x9bca] 1.3 [1.3.24278]
  custom_selector(cpu)    : cpu, Intel(R) OpenCL, Intel(R) Core(TM) i7-10710U CPU @ 1.10GHz 3.0 [2022.14.8.0.04_160000]
  custom_selector(acc)    : No device of requested type available. -1 (PI_ERRO...
```

**Comment by bader** [comment]
> FPGA emulator causes crashes on SYCL runtime initialization.

@yanfeng3721, do you know if this issue has been resolved?
If so, feel free to revert this change and enable FPGA emulator validation.

> BTW, will there be an oclfpgaemu CI testing similar as the current oclcpu CI tesing "Linux / OCL x64 LLVM Test Suite" in the future?

To be honest, I would rather run tests for FPGA features on FPGA emulator rather than the whole test suite. There is minor difference between OpenCL CPU and FPGA emulator, so running non-FPGA tests with OpenCL CPU should be good enough.

Tagging @pvchupin, @MrSidims and @AlexeySachkov to hear their opinions.

**Pull request merged**